app/policies/stripe_card_policy.rb

```diff
@@ -10,10 +10,14 @@ def shipping?
   end
 
   def freeze?
-    admin_or_manager? || organizer_and_cardholder?
+    admin_or_manager? || organizer_and_cardholder? || grantee?
   end
 
   def defrost?
+    if grantee? && !organizer? && (record.last_frozen_by != current_user)
```

Comment:
```suggestion
    if grantee? && !(admin_or_manager? || organizer_and_cardholder?) && (record.last_frozen_by != current_user)
```
I think otherwise readers could defrost cards they were the grantee for if an admin froze it. Yes, that's a weird edge case.

Reply:
No - this doesn't fix that bug. I think we should keep organizer at member level (we can rename it to member_and_cardholder?) because readers normally shouldn't be cardholders.

Reply:
Good question - so many edge cases. Do we currently cancel their cards when they're demoted?

Reply:
We don't. We really should imo.

Reply:
https://github.com/hackclub/hcb/issues/10512